checklists/network_appdelivery_checklist.en.json

```diff
@@ -342,7 +342,9 @@
             "guid": "a66f0fd8-2ca4-422e-8df3-235148127ca2",
             "id": "A01.54",
             "severity": "High",
+            "graph": "resources | where type == "microsoft.network/applicationgateways"| extend compliant = (properties['backendHttpSettingsCollection'][0]['properties']['port'] == "443") |where properties['backendHttpSettingsCollection'][0]['properties']['port'] == "443" |distinct id,name,compliant",
```

Comment:
* Please use the case-insensitive operator `=~` instead of `==`
* Please use single quotes for string comparison.
* Are we sure that the port is stored as a string and not as an integer?